package.json

```diff
@@ -27,10 +30,11 @@
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-contrib-watch": "^0.6.1",
-    "grunt-react": "^0.9.0",
+    "grunt-react": "^0.12.2",
     "grunt-webpack": "^1.0.8",
     "jsx-loader": "^0.12.2",
     "react": ">=0.12.1 <0.14",
+    "react-tools": "^0.13.3",
```

Comment:
This may not actually be required - I was experimenting with using `react-tools` directly before using `grunt-react`.